src/styles/game.css

```
@@ -5,6 +5,19 @@
   box-sizing: border-box;
 }
 
+/* Screen-reader only: visually hidden but accessible to assistive tech */
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.2.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)